### ISSUE TITLE
Respect environment variable LC_TIME for Date and Time Formats

### ISSUE THREAD

**Issue by dmarcoux** [Raymo111/i3lock-color#65]
Hey,

First of all, thank you for `i3lock-color`!

I have a small issue regarding the formatting of date and time. I expect the options `--timestr` and `--datestr` to respect `LC_TIME` as in their description, it refers to `strftime` (*See strftime(3) for a full list of format specifiers.*). `strftime` respects `LC_TIME` as stated in its man page (*The environment variables TZ and LC_TIME are used.*).

In the code, `LC_TIME` seems to never be taken into account and `strftime` doesn't seem to be used. I am not so sure about this as my knowledge of C is pretty limited.

My setup is the following if it can help:

```bash
export LC_COLLATE=en_US.UTF-8
export LC_CTYPE=en_US.UTF-8
export LC_TIME=fr_CA.UTF-8
export LANG=en_US.UTF-8
export LANGUAGE=en_US.UTF-8
```

```bash
$ i3lock --version
i3lock: version 2.10.1-color-non-git © 2010 Michael Stapelberg
```

**Comment by PandorasFox** [comment]
The locale checking is done [here](https://github.com/PandorasFox/i3lock-color/blob/master/i3lock.c#L1669) and just checks for LC_ALL/LC_CTYPE/LANG; it should be pretty simple to get it to check LC_TIME first.

I'll need to poke around a bit and see if locale affects anything else (I can't think of anything else that would, since there's not much localization in i3lock).

**Comment by PandorasFox** [comment]
should be fixed in 649a9d687d0b01c5a759c4d68920c8bdeae97ed2

**Comment by dmarcoux** [comment]
I will have a look in the next release. Thank you!
